src/core/server/plugins/plugin_context.test.ts

```diff
@@ -98,6 +98,7 @@ describe('createPluginInitializerContext', () => {
     expect(configObject).toStrictEqual({
       opensearchDashboards: {
         index: '.kibana',
+        dynamic_config_index: '.opensearch_dashboards_config',
```

Comment:
nitpick: `dynamic_config_index` feels long. I would be satisfied with `config_index`

Reply:
for the second point nvm: https://github.com/opensearch-project/security/blob/main/src/main/resources/static_config/static_roles.yml#L103

it actually is a lot better to do `.opensearch_dashboards_*` because kibana server role because it is all setup for that.

Reply:
Since point 2 and 3 are nvm, I just addressed point 1 to have a simple name.